src/NuGet.Core/NuGet.Protocol/Providers/RepositorySignatureResourceProvider.cs

```diff
@@ -25,11 +26,11 @@ public RepositorySignatureResourceProvider()
             var serviceIndex = await source.GetResourceAsync<ServiceIndexResourceV3>(token);
             if (serviceIndex != null)
             {
-                var repoSignUrl = serviceIndex.GetServiceEntryUri(ServiceTypes.RepositorySignatures);
+                var serviceEntry = serviceIndex.GetServiceEntries(ServiceTypes.RepositorySignatures).FirstOrDefault();
```

Comment:
>GetServiceEntries [](start = 48, length = 17)

I don't know how common this pattern is, but consider adding a `ServiceIndexResourceV3.GetServiceEntry` method like `GetServiceEntryUri`

Reply:
It's uncommon now, but if https://github.com/NuGet/Home/issues/8003 uncovers more use cases, I'll add a convenience method.